weather/jsonld-contexts/weather-compound.jsonld

```diff
@@ -0,0 +1,7 @@
+{
+    "@context": [
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/weather/jsonld-contexts/weather.jsonld",
+        "https://fiware.github.io/data-models/context.jsonld",

```

Comment:
did you have a look at the newer data models in the smart data models initiative?

this one still exists but is planned to disappear at some point.

Reply:
https://github.com/smart-data-models/dataModel.Weather

Reply:
and the context is here: https://github.com/smart-data-models/dataModel.Weather/blob/master/context.jsonld

Reply:
we can also contribute the missing terms, it avoids maintaining a specific subset

Reply:
Yes I did. My understanding was that the https://fiware.github.io/data-models/context.jsonld context contains all terms from fiware, i.e. covering the ones defined in https://github.com/smart-data-models/dataModel.Weather/blob/master/context.jsonld.
So, if the former will disappear, we will have to include several fiware contexts, right ? E.g. the ones for weather, agrifood, etc.

Reply:
The one on fiware.github.io is no longer maintained (`Last-Modified: Mon, 12 Apr 2021 12:38:25 GMT`), contexts are now under the Smart Data Models umbrella.

So, yes, we have to include the subdomains we need in our compound contexts (e.g., https://github.com/smart-data-models/dataModel.Weather/blob/master/context.jsonld).